packages/circuits/build.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+
+# default dir
+BUILD_DIR=./build
+PTAU=powersOfTau28_hez_final_20.ptau
+PTAU_PATH=$BUILD_DIR/$PTAU
+CIRCUIR_PATH=./circuits/qr_verify.circom
+CIRLIB_PATH=./node_modules
+R1CS_PATH=$BUILD_DIR/qr_verify.r1cs
+PARTIAL_ZKEYS_DIR=$BUILD_DIR/partial_zkeys
+
+CIRCOM_BIN_DIR=$HOME/.cargo/bin/circom
+
+# install circom and depenencies
+function install_deps() {
+
+    if [ ! -d $BUILD_DIR ]; then
+        mkdir -p $BUILD_DIR
+    fi
+
+    echo "Install circom"
+    cd $BUILD_DIR
+    if [ ! -f $CIRCOM_BIN_DIR ]; then
+        git clone https://github.com/iden3/circom.git
+        cd circom
+        cargo build --release
+        cargo install --path circom
+        echo "Installed circom!"
+    else 
+        echo "Circom already install... Skip this action!"
+    fi 
+
+    echo "Download power of tau...."
+    cd $BUILD_DIR    
+    if [ ! -f $PTAU_PATH ]; then
+        wget https://hermez.s3-eu-west-1.amazonaws.com/$PTAU
+        echo "Finished download!"
+    else 
+        echo "Powers of tau file already downloaded... Skip download action!"
+    fi 
+
+    echo "Finished install deps!!!!"
+}
+
+# trusted setup for development
+# DON'T USE IT IN PRODUCT
+function dev_trusted_setup() {
+    echo "Starting setup...!"
+
+    if [ ! -d $PARTIAL_ZKEYS_DIR ]; then
+        mkdir -p $PARTIAL_ZKEYS_DIR
+    fi
+
+    echo "TRUSTED SETUP FOR DEVELOPMENT - PLEASE, DON'T USE IT IN PRODUCT!!!!"
+
+    circom ./circuits/qr_verify.circom  --r1cs --wasm -o $BUILD_DIR -l ./node_modules
+
+    yarn add snarkjs@git+https://github.com/vb7401/snarkjs.git#24981febe8826b6ab76ae4d76cf7f9142919d2b8
+	yarn
+
+
+    NODE_OPTIONS=--max-old-space-size=8192 \
+	node ./node_modules/.bin/snarkjs groth16 setup $R1CS_PATH $PTAU_PATH $PARTIAL_ZKEYS_DIR/circuit_0000.zkey
+
+    echo "test random" | NODE_OPTIONS='--max-old-space-size=8192' \
+	node ./node_modules/.bin/snarkjs zkey contribute $PARTIAL_ZKEYS_DIR/circuit_0000.zkey $PARTIAL_ZKEYS_DIR/circuit_final.zkey --name="1st Contributor Name" -v 
+
+    yarn remove snarkjs
+    yarn add snarkjs@0.7.0
+    yarn
+
+    NODE_OPTIONS=--max-old-space-size=8192 \
```

Comment:
Why do we need to run the same command 2 times with different versions of `snarkjs`?

Reply:
I removed those lines!